gdp-src-build/conf/templates/silk.bblayers.conf

```diff
@@ -0,0 +1,7 @@
+include templates/bblayers.inc
+
+# porter specific layer configuration
+BBLAYERS += " \
+  ${TOPDIR}/../meta-renesas \
+  ${TOPDIR}/../meta-renesas/meta-rcar-gen2 \
```

Comment:
This appears to be a change introduced by the GDP maintainers in their structural changes. The OE multimedia layer was present in the old structure (see porter branch http://git.projects.genivi.org/?p=genivi-demo-platform.git;a=blob;f=gdp-src-build/conf/bblayers.conf;h=dc8c0d4930044434b8ef6d9906b15154692704ce;hb=refs/heads/porter).

I can see how it might have occurred. During the early days of GDP-ivi9 that used my experimental ivi9 branches I temporarily removed the genivi-demo-platform.bbappend that pulled in the multimedia packages as GDP did not make use of multimedia acceleration and at that point I had not fully tested it. I guess one of the maintainers used what was needed at the time rather than the confs from the porter branch and hence it was moving. Re-enabling multimedia  was one of the improvements I was seeking to introduce with the Silk/Koelsch update I pushed to the ML before the Paris AMM.There it was ok because the confs were correct.